src/crypto/sha256.cpp

```diff
@@ -615,12 +615,9 @@ std::string SHA256AutoDetect()
 #endif
 
     if (have_sse4) {
```

Comment:
Move the `if` statement inside of the `#if defined(ENABLE_SSE41) && !defined(BUILD_BITCOIN_INTERNAL)` to remove the possibility of an empty if statement.